### PR TITLE
FS Settings

### PR DESCRIPTION
**Pull request by grogsy**
PR for #30 

Untitled

All the buttons and forms should be fully functional. I added a few more api routes for the client to interface with the backend. The user can now modify their subscription email, track multiple companies in addition to the one they registered with, and remove companies they no longer want to track. Clicking on logout sends the user to the login page, and clicking the go back button sends the user to main page.

### CODE REVIEW

File: client/src/Components/settings/forms/AddCompanyForm.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        variant="outlined"
+        className={classes.formField}
+        label="Company Name"
+        value={input}
+        onChange={(e) => setInput(e.target.value)}
+        InputLabelProps={{
+          style: {
+            fontWeight: "bold",
+          },
+        }}
```
Comment (jt5301) [inline client/src/Components/settings/forms/AddCompanyForm.js:31]: I have a custom textfield component in the components folder that I used for the login/signup pages. Is that usable here?

Comment (grogsy) [reply client/src/Components/settings/forms/AddCompanyForm.js:31]: It should be. Don't know why I didn't think of that sooner. I'm going to try it out and if no probs will be using your component

Comment (grogsy) [reply client/src/Components/settings/forms/AddCompanyForm.js:31]: I think when I started out with the ticket I didn't know how to proceed so I might've done some base components from scratch

Comment (grogsy) [reply client/src/Components/settings/forms/AddCompanyForm.js:31]: Actually, to get the input text to fit the spec I used some extra props asked by `TextField`(InputLabelProps & InputProps), so maybe not yet :/

Comment (jt5301) [reply client/src/Components/settings/forms/AddCompanyForm.js:31]: I see. No worries! Just curious. I think there should be a way to modify the component I have to fit needs, but if it's easier to write a new one that makes sense.

**Comment by jt5301** [comment]
I can't add comments on the code itself for some reason, but everything looks good to me! The only thing is that the right panel is still a shade of blue, I think it's just supposed to be set white.
Also, when i start resizing the screen a little, this "Go Back" button gets cut off / warped into two lines. Would wrapping it in another div help?

Screen Shot 2020-07-14 at 4 56 55 PM

**Comment by grogsy** [comment]
Oh right I forgot what color its supposed to be. I set it back to white now. As for the button I set a style as `{whiteSpace: 'nowrap'}` which prevents it from squeezing down

**Review by jt5301 — APPROVED** [review]

**Pull request merged**